docs/ci-cd-integration/deploy-from-github.md

```diff
@@ -131,7 +131,20 @@ You will need to add the following information into GitHub Secrets:
 You can add GitHub Secrets using the instructions [here](#add-the-github-secrets)
 
 #### Add the Captain Definition File
-Add the `captain-definition` file described [here](#add-files-to-project)
+Add the `captain-definition` file described [here](#add-files-to-project). If are going to be using the github action below, the `captain-definition` contents should be:
```

Comment:
I just realized that this was a mistake to add this section and it should be fully deleted.

The "Alternative method" that is in the doc uses a simple image deploy, and it does not use `captain-definition` file at all. That file and its content is fully ignored in this approach.

Let's remove this `captain-definition` section altogether. 

![image](https://github.com/caprover/caprover-website/assets/10582082/1588f40c-f4b4-4661-98db-2ec714ebd91c)

Reply:
That's odd, I was trying to deploy without a `captain-definition` and it wasn't working, the entires in the version history were all blank until I added the `captain-definition` file. But now that you said that I removed it and tried again and it worked without it. 🤷 

Will remove it!